src/tds/codec/rpc_request.rs

```diff
@@ -10,19 +10,19 @@ pub struct TokenRpcRequest<'a> {
     proc_id: RpcProcIdValue<'a>,
     flags: RpcOptionFlags,
     params: Vec<RpcParam<'a>>,
-    transaction_id: u64,
+    transaction_desc: [u8; 8],
```

Comment:
Is that change motivated by endianness concerns? I can see we write that with `put_slice()` instead of `put_u64_le()` now.

Reply:
I just do whatever the JDBC driver does. We always just write bytes to the message header, so there is no reason to convert that to an arbitrary number.